opendnp3/APL/TimeBoost.cpp

```diff
@@ -43,6 +43,9 @@ TimeBoost::TimeBoost(millis_t aTimeMS)
 TimeBoost::TimeBoost(ptime aTime) :
 	mTime(aTime)
 {
+        if(GetValueMS() < 0) {
+            SetTo(GetCurrentTimeInMsec());
+        }
```

Comment:
Make sure you follow the same indention format as the file. Tabs

Reply:
these checks are all safe and unknown data to cover it. there is no harm. let us retest again by removing it.